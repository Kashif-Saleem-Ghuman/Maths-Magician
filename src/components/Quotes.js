import React from 'react';

export default function Quotes() {
  return (
    <>
      <section className="quotes-wrapper">
        <h1 className="quote-h1">
          &quot;Mathematics is not about numbers, equations, computations,
          or algorithms: it is about understanding.&quot;
        </h1>
        <span className="quotes-span">- William Paul Thurston</span>
      </section>
      <footer>
        <span className="footer-text">Created by Kashif Saleem</span>
        <a className="github" target="_blank" rel="noreferrer" href="https://github.com/Kashif-Saleem-Ghuman">Github</a>
      </footer>
    </>
  );
}
