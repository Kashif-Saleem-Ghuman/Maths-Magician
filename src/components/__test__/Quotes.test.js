import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../Quotes';

it('renders when there is quote', () => {
  const quotes = renderer.create(<Quotes />).toJSON();
  expect(quotes).toMatchSnapshot();
});
