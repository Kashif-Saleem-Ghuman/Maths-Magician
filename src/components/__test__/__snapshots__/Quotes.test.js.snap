// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there is quote 1`] = `
Array [
  <section
    className="quotes-wrapper"
  >
    <h1
      className="quote-h1"
    >
      "Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding."
    </h1>
    <span
      className="quotes-span"
    >
      - William Paul Thurston
    </span>
  </section>,
  <footer>
    <span
      className="footer-text"
    >
      Created by Kashif Saleem
    </span>
    <a
      className="github"
      href="https://github.com/Kashif-Saleem-Ghuman"
      rel="noreferrer"
      target="_blank"
    >
      Github
    </a>
  </footer>,
]
`;
